style/rails/README.md

```diff
@@ -19,9 +19,11 @@ Rails
 * Use `def self.method`, not the `scope :method` DSL.
 * Use the default `render 'partial'` syntax over `render partial: 'partial'`.
 * Use `link_to` for GET requests, and `button_to` for other HTTP verbs.
+* Prefer ERb or [Slim][slim-lang] over HAML for templating
```

Comment:
Is ERB supposed to be all caps? http://ruby-doc.org/stdlib-2.2.3/libdoc/erb/rdoc/ERB.html